hs/src/Reach/Connector/ALGO.hs

```diff
@@ -1588,6 +1615,7 @@ cloop which (C_Loop at svs vars body) = recordWhich which $ do
 ch :: LT.Text -> Int -> CHandler -> App ()
 ch _ _ (C_Loop {}) = return ()
 ch afterLab which (C_Handler at int from prev svs msg_ timev secsv body) = recordWhich which $ do
+  resetNewToks
```

Comment:
I am anxious about this style of one global box that gets reset at the start of a handler. I'd prefer if this was part of `recordWhich` (which uses local state for the which and duplicates the IORef of the txn count). I think it should fit there, maybe `dupeNewToks`?